### ISSUE TITLE
Failed downloading jce_policy-6.zip

### ISSUE THREAD

**Issue by maksimu** [flexiondotorg/oab-java6#108]
Here are my steps:

```
ubuntu@ip-10-124-173-94:~$ chmod +x oab-java.sh
ubuntu@ip-10-124-173-94:~$ sudo ./oab-java.sh 
oab-java.sh v0.2.6 - Create a local 'apt' repository for Sun Java 6 and/or Oracle Java 7 packages.

Copyright (c) Martin Wimpress, http://flexion.org. MIT License

By running this script to download Java you acknowledge that you have
read and accepted the terms of the Oracle end user license agreement.

* http://www.oracle.com/technetwork/java/javase/terms/license/

If you want to see what this is script is doing while it is running then execute
the following from another shell:

  tail -f /home/ubuntu/oab-java.sh.log

 [x] Installing Java build requirements success
 [x] Making build directories success
 [x] Removing clones of https://github.com/rraptorr/sun-java6 success
 [x] Cloning https://github.com/rraptorr/sun-java6 success
 [x] Checking out v6.45-1 success
 [x] Getting Java SE download page success
 [x] Getting current release download page success
 [x] Getting previous releases download page success
 [x] Downloading jdk-6u45-linux-i586.bin : 68.47 MB success
 [x] Symlinking jdk-6u45-linux-i586.bin success
 [x] Downloading jdk-6u45-linux-x64.bin : 68.75 MB success
 [x] Symlinking jdk-6u45-linux-x64.bin success
 [x] Getting Java Cryptography Extension download page success
 [x] Downloading jce_policy-6.zip :  failed
 [i] Showing the last 5 lines from the logfile (/home/ubuntu/oab-java.sh.log)...
 [x] Downloading jce_policy-6.zip :    --2013-07-05 23:48:21--  http://jce_policy-6.zip/
Resolving jce_policy-6.zip (jce_policy-6.zip)... failed: Name or service not known.
wget: unable to resolve host address `jce_policy-6.zip'
15529's retcode: 4
failed
```

**Comment by shuhaowu** [comment]
+1 I can confirm this.

**Comment by smashegg** [comment]
+1 got the same :(

**Comment by shuhaowu** [comment]
It looks like Oracle changed their web page... >_>

**Comment by Boboss74** [comment]
Duplicate of #101.

**Comment by flexiondotorg** [comment]
Merged pull-requuest #111.
